_posts/en/newsletters/2018-11-27-newsletter.md

```diff
@@ -0,0 +1,210 @@
+  fee bump even a small transaction that signals opt-in Replace-by-Fee
+  (RBF).  (Transaction pinning can create problems for protocols such as
+  LN where security depends on some transactions confirming within a
+  certain period of time.)
+
+- [What makes batch verification of Schnorr signatures effective?][bse
+  80702] Pieter Wuille provides a simple explanation for how it's
+  possible to do several multiplication operations simultaneously on an
+  elliptic curve.  This can be significantly faster than doing single
+  multiplication in series, allowing multiple signatures to be verified
+  together faster than they could be individually verified.
+
+## Notable code changes
+
+*Notable code changes this week in [Bitcoin Core][core commits],
+[LND][lnd commits], [C-lightning][cl commits], and [libsecp256k1][secp
+commits].*
+
+{% include linkers/github-log.md
+  refname="core commits"
+  repo="bitcoin/bitcoin"
+  start="35739976c1d9ad250ece573980c57e7e7976ae23"
+  end="a7dc03223e915d7afb30498fe5faa12b5402f7d8"
+%}
+{% include linkers/github-log.md
+  refname="lnd commits"
+  repo="lightningnetwork/lnd"
+  start="4da1c867c3209dab4e4a824b73d89fc38b616b37"
+  end="8924d8fb20eb2abfd9cc93c6cc7eb6951184cb88"
+%}
+{% include linkers/github-log.md
+  refname="cl commits"
+  repo="ElementsProject/lightning"
+  start="d5aaa11373cc6759f9f894a1daf7fb88d0834bc9"
+  end="95e47cdac298b8e534feb073c70da004c08b3e93"
+%}
+{% include linkers/github-log.md
+  refname="secp commits"
+  repo="bitcoin-core/secp256k1"
+  start="314a61d72474aa29ff4afba8472553ad91d88e9d"
+  end="314a61d72474aa29ff4afba8472553ad91d88e9d"
+%}
+
+- [Bitcoin Core #14708][] prints a warning when unrecognized section
+  names are used in the `bitcoin.conf` configuration file.  For example,
+  if you create the following configuration file using the name
+  `testnet` instead of the correct name `test`, Bitcoin Core would
+  previously silently ignore the testnet options.  This merged PR causes
+  it to print a notice: "Warning: Section [testnet] is not recognized."
+
+
+    ```toml
+    [testnet]
+    txindex = 1
+    ```
+
+- [C-Lightning #2087][] adds new fields to the results of the `getinfo` RPC for
+  the number of the node's peers, number of pending channels, number of
+  active channels, and number of inactive channels.  This now matches
+  information displayed by LND's `getinfo` RPC.
+
+- [C-Lightning #2096][] strips the text `lightning:` prefixed to a
+  [BOLT11][] invoice before attempting to process it.  This text is
+  sometimes added so that LN wallets can register for it as URI
+  handlers.  The prefix text will be striped if it is all lowercase or
+  all uppercase (but not mixed case) per the [BIP173][] bech32
+  specification.
+
+- C-Lightning [#2081][c-lightning #2081] and [#2092][c-lightning #2092]
+  fix a problem with running multiple RPC commands in parallel.  As a
+  user-visible change, lightningd now adds a double newline (`\n\n`)
```

Comment:
Should be `lightningd` instead fo lightningd to make it consistent with how `bitcoind` is written below? :-)